pkg/reconciler/scheduling/placement/placement_indexes.go

```diff
@@ -37,15 +36,15 @@ func indexByWorkspace(obj interface{}) ([]string, error) {
 	return []string{lcluster.String()}, nil
 }
 
-func indexByNegotiationWorkspace(obj interface{}) ([]string, error) {
-	ns, ok := obj.(*corev1.Namespace)
+func indexByLoactionWorkspace(obj interface{}) ([]string, error) {
```

Comment:
```suggestion
func indexByLocationWorkspace(obj interface{}) ([]string, error) {
```

Reply:
But this is strange. Do you mean`bySelectedLocation` or `byLocationWorkspace`?

Reply:
think again, it should be byLocationWorkspace which reads locationworkspace in spec I think.